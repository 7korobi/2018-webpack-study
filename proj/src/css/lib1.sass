
body
  background-color: blue

.content
  code
    display: flex
