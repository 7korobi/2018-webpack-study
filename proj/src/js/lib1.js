
export default {
  z: 1
}