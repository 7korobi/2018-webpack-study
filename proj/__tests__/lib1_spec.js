import lib1 from '../src/js/lib1';

describe("lib1", ()=> {
  test('snapshot', ()=> {
    expect(lib1).toMatchSnapshot()
  })
})

