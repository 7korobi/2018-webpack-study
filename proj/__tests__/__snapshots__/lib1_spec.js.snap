// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib1 snapshot 1`] = `
Object {
  "z": 1,
}
`;
